Cargo.toml

```diff
@@ -42,6 +42,6 @@ opt-level = "s"          # <-
 overflow-checks = false  # <-
 
 [patch.crates-io]
-embassy-executor = { version = "0.3", git = "https://github.com/embassy-rs/embassy", rev = "5f7cd82" }
-embassy-rp = { version = "0.1", git = "https://github.com/embassy-rs/embassy", rev = "5f7cd82" }
-embassy-time = { version = "0.2", git = "https://github.com/embassy-rs/embassy", rev = "5f7cd82" }
+embassy-executor = { version = "0.4", git = "https://github.com/embassy-rs/embassy", rev = "df0f41c41c685c28965942bf6e6c67d8ca166d40" }
```

Comment:
I think this would need to be bumped somewhere in a `[dependency]` section, and not just the patch section?